ansible/playbooks/apache.yml

```
@@ -0,0 +1,102 @@
+    - name: Install Apache2
+      apt:
+        name: apache2
+        state: present
+        
+    - name: Ensure Apache2 is started and enabled
+      service:
+        name: apache2
+        state: started
+        enabled: yes
+      ignore_errors: yes  # In case enable fails in container
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)